src/components/foodbox/Foodbox.jsx

```diff
@@ -0,0 +1,39 @@
+
+import './Foodbox.css';
+
+
+export default function FoodBox({ name, calories, image, quantity }) {
```

Comment:
Este componente que sea tipo **class** para que pueda tener estado

Reply:
tampoco hace falta sacar la variable **quantity** de las **props**, ya que no se la estáis pasando, ni la vais a necesitar

Reply:
Este componente debe recibir entre las **props** una función que modifique el estado del padre, para poder seleccionar **foods**